profile/README.md

```diff
@@ -0,0 +1,90 @@
+# Rust Bitcoin Community
+
+A Series of projects to implement various Bitcoin protocols in Rust.
+
+## Core Crates
+
+The "core" crates developed and maintained collectively by the `rust-bitcoin` devs.
+
+For these crates we hold ourselves to high standards relating to security, testing, release
+schedule, API design, and coding style. Our current focus is on stabilising these crates [0].
+
+Name | Crate | Repository
+---|---|---
+base58 | [base58ck](https://crates.io/crates/base58ck) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+bitcoin | [bitcoin](https://crates.io/crates/bitcoin) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+bitcoinconsensus | [bitcoinconsensus](https://crates.io/crates/bitcoinconsensus) | [rust-bitcoinconsensus](https://github.com/rust-bitcoin/rust-bitcoinconsensus)
+bech32 | [bech32](https://crates.io/crates/bech32) | [rust-bech32](https://github.com/rust-bitcoin/rust-bech32)
+hashes | [bitcoin_hashes](https://crates.io/crates/bitcoin_hashes) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+hex | [hex-conservative](https://crates.io/crates/hex-conservative) | [hex-conservative](https://github.com/rust-bitcoin/hex-conservative)
+internals | [bitcoin-internals](https://crates.io/crates/bitcoin-internals) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+io | [bitcoin-io](https://crates.io/crates/bitcoin-io) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+ordered | [ordered](https://crates.io/crates/ordered) | [rust-ordered](https://github.com/rust-bitcoin/rust-ordered)
+secp256k1 | [secp256k1](https://crates.io/crates/secp256k1) | [rust-secp256k1](https://github.com/rust-bitcoin/rust-secp256k1)
+units | [bitcoin-units](https://crates.io/crates/bitcoin-units) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+
+We aim to make usage of these crates secure, ergonomic, and intuitive. We try to enable users of
+these crates to follow software engineering best practices e.g., semver compliance.
+
+If we fall short of these stated aims, or can in anyway improve our support of Bitcoin free software
+projects, please raise issues or contact maintainers directly.
```

Comment:
We could link to the https://github.com/rust-bitcoin/rust-bitcoin/blob/master/SECURITY.md

And also we could do something like enumerating the maintainers, as in
https://github.com/rust-bitcoin/rust-bitcoin/blob/47527abd536cbc30e47bce127cc9d2b7c8cc0b98/CONTRIBUTING.md?plain=1#L142-L151
I know that there's a risk this gets outdated...

Reply:
Added security link. The maintainer list is crate specific, I'd prefer not to add any names to the org readme. @apoelstra please weigh in if you think differently.

Reply:
Yeah, agreed, let's leave names out of this doc for now.